### PR TITLE
GAV tag for Java ops

### LINKED ISSUE #647: add gav tag for all java operations

### PR DESCRIPTION
**Pull request by mihaitusa**
fixes #647

### CODE REVIEW

File: content/io/cloudslang/cloud/amazon_aws/images/reset_launch_permissions_on_image_in_region.sl [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #!               Note: The productCodes attribute can't be reset.
 #! @input provider: Cloud provider on which the instance is - Default: 'amazon'
 #! @input endpoint: Endpoint to which first request will be sent - Default: 'https://ec2.amazonaws.com'
-#! @input identity: optional - Amazon Access Key ID
+#! @input identity: optional - Amazo Access Key ID
```
Comment (gituser173) [inline content/io/cloudslang/cloud/amazon_aws/images/reset_launch_permissions_on_image_in_region.sl]: typo?

Comment (mihaitusa) [inline content/io/cloudslang/cloud/amazon_aws/images/reset_launch_permissions_on_image_in_region.sl]: @gituser173 You are right... is typo... I will take care of it. Good catch!

**Comment by mihaitusa** [comment]
@meshipeer We will wait a little bit for Genadi answer related to gav/dependencies

**Comment by Bonczidai** [comment]
:fireworks:

**Pull request merged**